package.json

```
@@ -28,6 +28,8 @@
   },
   "dependencies": {
     "react": "16.6.3",
+    "react-bootstrap": "^0.32.4",
```

In apps, your should favour fixed dependency versions so the modules behave as intended after each install. Note: this is different when you're creating a module.

```suggestion
    "react-bootstrap": "0.32.4",
```

---

Yeah it makes sense to set them all to a specific version, will update that. Right, a custom module would grow in versioning one develops it.